### ISSUE TITLE
[Bug] br_spaces parameter doesn't work

### ISSUE THREAD

**Issue by Kristinita** [markdownlint/markdownlint#165]
### 1. Summary

`br_spaces` parameter for `MDOO9` rule doesn't work for me.

### 2. Settings

+ My `Nas-Izu.md` file — **<https://github.com/Kristinita/KristinitaPelican/blob/master/content/Giologica/Nas-Izu.md>**

Trailing whitespaces in a file:

![Trailing whitespaces](http://i.imgur.com/1AM4rTD.png)

+ My `.mdlrc` file — **<https://github.com/Kristinita/KristinitaPelican/blob/master/.mdlrc>**

```ruby
style "#{File.dirname(__FILE__)}/SashaMarkdownLint.rb"
```

+ My `SashaMarkdownLint.rb` file:

```ruby
all

rule 'MD009', :br_spaces => 3 # Lint if more than 3 trailing spaces

exclude_rule 'MD010' # I use tabs instead of spaces
exclude_rule 'MD013' # Line length
exclude_rule 'MD033' # Inline HTML support
exclude_rule 'MD041' # In Pelican first line — is Metadata, not header
```

### 3. Expected behavior

+ if trailing whitespace number ≥ 3, I get errors in a terminal,
+ if trailing whitespace number ≤ 3, I don't get errors in a terminal.

### 4. Actual behavior

If trailing whitespace number > 0, I get errors in a terminal:

```ruby
D:\Kristinita\content\Giologica>mdl Nas-Izu.md
Nas-Izu.md:5: MD009 Trailing spaces
Nas-Izu.md:6: MD009 Trailing spaces
Nas-Izu.md:7: MD009 Trailing spaces
Nas-Izu.md:8: MD009 Trailing spaces
Nas-Izu.md:26: MD034 Bare URL used
```

### 5. Environment

**Operating system and version:**
Windows 10 Enterprise LTSB 64-bit EN
**Ruby:**
ruby 2.3.1p112 (2016-04-26 revision 54768) [x64-mingw32]
**MarkownLinter:**
0.4.0

Thanks.

**Comment by schwarzie2478** [comment]
I don't know if this was fixed already or not, but when I used this linter in VS Code,  it worked for me with 2 trailing spaces.   config file:  
`{
    "MD009": {"br_spaces": 2},
    "MD013": false
}`

**Comment by DavidAnson** [comment]
If you are referring to https://github.com/DavidAnson/vscode-markdownlint, it uses a different implementation. It is possible for slight behavioral differences to be present; this issue may be valid for the Ruby implementation.

**Comment by Kristinita** [comment]
### 1. Status

Works as expected. It my, not Markdownlint error.

### 2. Cause of error

I'm not Ruby programmer and I misunderstood `=>` operator. [**It not**](https://stackoverflow.com/a/4663089/5951529) `more or equal than <number>` in Ruby.

### 3. Examples of expected behavior

**If** in `styles.rb`:

```ruby
rule 'MD009', :br_spaces => 2
```

+ 2 trailing spaces in line — no output.
+ 1, 3 or 4 — warnings (or is it errors) in output.

**Else**:

```ruby
rule 'MD009', :br_spaces => 3
```

+ 3 trailing spaces in line — no output.
+ 1, 2 or 4 — warnings (or is it errors) in output.

It would be nice, if this point were better covered in the documentation.

Thanks.
